knowledge-hub: add AI factory topic (#2494)
canonical/canonical.com: templates/knowledge/data-and-ai-ml/what-is-an-ai-factory.md

        type: "button"
        variant: "positive"
      - text: "Get in touch"
        url: "https://canonical.com/solutions/ai/infrastructure#get-in-touch"
        type: "button"
  blog:
    title: "Latest from our blog"
    id: 4062
---
{% from "macros/_macros-text-list.jinja" import text_list_kh %}
{% from "macros/_macros-image.jinja" import image_kh %}

An AI factory is a dedicated infrastructure environment optimized for the large-scale production, fine-tuning, and deployment of AI models. AI factories make AI workloads repeatable, scalable, and cost-effective.

## What is the difference between an AI factory and a data center?

Traditional data centers are built for general-purpose workloads such as web applications, email systems, and batch processing. AI factories are designed specifically for AI workloads, which changes their compute architecture, networking model, and energy profile.

### Compute architecture

Traditional data centers rely mainly on CPUs, while AI factories use specialized accelerators for parallel AI workloads:

{{ text_list_kh(
  items=[
    "Graphics Processing Units (GPUs)",
    "Tensor Processing Units (TPUs)",

>     "Application-specific integrated circuits (ASICs), including LPUs and DPUs"

Reviewer: ```suggestion
    "Application-specific Integrated Circuits (ASICs) such as Language Processing Units (LPUs)"
```

]) }}

These accelerators are designed for massive parallel processing, enabling high-volume, simultaneous calculations for model training and inference.

### Networking

> AI factories require much lower latency between nodes than traditional data centers. Large models typically run across many nodes, so tail latency can stall the entire cluster.

Reviewer: ```suggestion
AI factories require much lower latency between nodes than traditional data centers. Large models typically run across many nodes, so "tail latency" can stall the entire cluster.
```

> Traditional data center traffic is often north-south (client to server). AI factory traffic is predominantly east-west (accelerator to accelerator), with high-bandwidth fabrics such as InfiniBand or highly optimized Ethernet.

Reviewer: ```suggestion
Network traffic in a traditional data center typically moves in a “north-south” pattern between the user and the server, using standard Ethernet. In AI factories, traffic predominantly moves between GPUs in an “east-west” pattern, and the infrastructure relies on high-bandwidth fabrics such as Infiniband or specially optimized Ethernet that can rapidly move large data volumes without congestion.
```

### Energy requirements

> AI factories run at higher power density and require stronger cooling systems because accelerator-heavy workloads are compute-intensive.

Reviewer: ```suggestion
AI factories have higher power densities and cooling requirements than traditional data centers due to the compute-intensive nature of the AI accelerators. To address this challenge, AI factories are typically designed with specially optimized cooling systems to improve operational costs.
```

[Learn how to build an AI factory with our guide to AI infrastructure ›](https://ubuntu.com/engage/open-source-ai-infrastructure)

## What can you do with an AI factory?

AI factories support the full AI lifecycle and industrialize model development and delivery.

### Model training and fine-tuning

> Train new models or fine-tune existing ones at scale to optimize for domain-specific tasks and latency requirements.

Reviewer: ```suggestion
You can train new AI models with your data, or fine-tune existing models at scale to optimize them for your specific use cases. This gives you AI models that can deliver intelligence in real-time based on their training data, making this approach ideal when latency is critical. 
```

### Retrieval augmented generation (RAG)

> Augment pretrained models with current data at runtime so they can answer with context they were not originally trained on.

Reviewer: please match the copy doc content for the following sections.

### Operational efficiency and scalability

Integrated tooling and automation reduce manual bottlenecks and support growth in model and data scale.

### Security and compliance

Centralized governance supports data lineage, model versioning, and access controls for regulated environments.

## Why are private AI factories becoming popular?

Organizations increasingly deploy AI factories on private infrastructure for:

{{ text_list_kh(
  items=[
    "<strong>Sovereignty:</strong> Keeping AI infrastructure on-premises helps satisfy compliance and data control requirements.",
    "<strong>Cost predictability:</strong> Constant, high utilization can make owned infrastructure more economical than hourly cloud accelerator pricing."
]) }}

[Learn more about cloud sovereignty in our guide ›](https://ubuntu.com/engage/sovereign-cloud-guide)

## What are common AI factory use cases?

AI factories can be used anywhere organizations need continuous, large-scale intelligence generation, including:

{{ text_list_kh(
  items=[
    "Manufacturing: predictive maintenance, process automation, quality control",

>     "Autonomous vehicles: model training, validation, and fleet orchestration",

Reviewer: ```suggestion
    "Autonomous vehicles: training autonomous driving systems, digital twin validation, and fleet orchestration",
```

>     "Healthcare: drug discovery, diagnostics, and personalization",

Reviewer: Doesn't seem to be aligned with the copy doc:
```
Healthcare
- Drug discovery
- Medicine personalization
- Patient diagnostics
```

>     "Financial services: fraud detection, risk modeling, and forecasting",

Reviewer: same here:
```
Financial services
- Fraud detection
- Risk assessment
- Market simulation and forecasting
```

>     "Telecommunications: network orchestration and threat detection",

Reviewer: ```suggestion
    "Telecommunications: network orchestration, threat detection, and customer support",
```

>     "Enterprise productivity: private data analysis and AI-powered internal applications"

Reviewer: Doesn't seem to be aligned with the copy doc:
```
LLM as a service
- Productivity workflows
- Large-scale private data analysis
- Enterprise apps
```

    "<strong>Cost predictability</strong> - AI factories are designed to run 24/7, so it becomes more cost-effective for an organization to own the AI infrastructure itself, rather than pay hourly GPU rates to public cloud providers."
]) }}

[Learn more about cloud sovereignty in our guide ›](https://ubuntu.com/engage/sovereign-cloud-guide)

## What are the industry use cases for an AI factory?

AI factories can be utilized in any industry for any use case where organizations need to continuously generate actionable intelligence and agentic experiences at scale. Examples include:

{{ text_list_kh(
  items=[
    "Manufacturing: Predictive maintenance; Process automation; Quality control",
    "Autonomous vehicles: Training autonomous driving systems; Digital twin validation; Fleet orchestration",
    "Healthcare: Drug discovery; Medicine personalization; Patient diagnostics",
    "Financial services: Fraud detection; Risk assessment; Market simulation and forecasting",
    "Telecommunications: Network orchestration; Threat detection; Customer support LLM as a service",
    "Productivity workflows: Large-scale private data analysis; Enterprise apps"

> ]) }}

Reviewer: ```suggestion

### Manufacturing

{{ text_list_kh(items=[
  "Predictive maintenance",
  "Process automation",
  "Quality control"
]) }}

### Autonomous vehicles

{{ text_list_kh(items=[
  "Training autonomous driving systems",
  "Digital twin validation",
  "Fleet orchestration"
]) }}

### Healthcare

{{ text_list_kh(items=[
  "Drug discovery",
  "Medicine personalization",
  "Patient diagnostics"
]) }}

### Financial services

{{ text_list_kh(items=[
  "Fraud detection",
  "Risk assessment",
  "Market simulation and forecasting"
]) }}

### Telecommunications

{{ text_list_kh(items=[
  "Network orchestration",
  "Threat detection",
  "Customer support LLM as a service"
]) }}

### Productivity workflows

{{ text_list_kh(items=[
  "Large-scale private data analysis",
  "Enterprise apps"
]) }}

```
